src/js/core/widget/core/Appbar.js

```diff
@@ -351,6 +368,51 @@
 				expandedTitle.style.opacity = expandLevel;
 			};
 
+			/**
+			 * Expand AppBar
+			 * @method expand
+			 * @member ns.widget.core.Appbar
+			 */
+			prototype.expand = function () {
+				var self = this;
```

Comment:
How about defining element as  well, we are going to use it twice.

```
var self = this,
    element = self.element;
```

Reply:
You are right

Reply:
Done.